cextern/trim_cfitsio.sh

```diff
@@ -29,7 +31,7 @@ rm -f cfitsio/docs/*.ps
 rm -f cfitsio/docs/*.pdf
 rm -f cfitsio/docs/*.doc
 rm -f cfitsio/docs/*.toc
-rm -f cfitsio/[^L]*.*
+rm -rf cfitsio/[^L]*.*
```

Comment:
I'm surprised by the need for this change - this means that in the new `cfitsio` library there is a directory with a `.` in its name. Out of curiosity, what is it?

Reply:
I guess I should wait till this is answered before merging?

Reply:
For me it was `cfitsio/cfitsio.xcodeproj`, which appears to be generated when building (on OSX for me). Maybe we should just explicitly remove that file rather than using `-rf` here?

Reply:
So, that must be a new directory. I think I'd prefer removing it explicitly, but honestly I don't really know why...